meerket/meerket-domain/src/main/java/org/j1p5/domain/comment/entity/CommentEntity.java

```diff
@@ -43,4 +45,8 @@ public class CommentEntity extends BaseEntity {
     @Column(name = "comment_status", nullable = false)
     @Enumerated(EnumType.STRING)
     private CommentStatus status;
+
+    public void update(String content){
```

Comment:
```suggestion
    public void updateContent(String content){
```
로 명확한 네이밍 수정이 좋을 것 같아요!